src/experiment-configuration-requestor.ts

```diff
@@ -25,6 +25,10 @@ export default class ExperimentConfigurationRequestor {
 
   async fetchAndStoreConfigurations(): Promise<Record<string, IExperimentConfiguration>> {
     const responseData = await this.httpClient.get<IRandomizedAssignmentConfig>(RAC_ENDPOINT);
+    if (!responseData) {
+      return {};
+    }
```

Comment:
throw an error here?

Reply:
Perhaps. In the Ruby and Python SDKs, we log to stdout: https://github.com/Eppo-exp/ruby-sdk/blob/main/lib/configuration_requestor.rb#L91 and https://github.com/Eppo-exp/python-sdk/blob/main/eppo_client/configuration_requestor.py#L60

Assuming that this will most likely pipe to CloudWatch logs or something similar to help customers troubleshoot if fetching the config isn't working

Reply:
I think just logging it will be fine.